web/modules/custom/vehicle_custom/src/Service/VehicleLookupService.php

```diff
@@ -35,19 +35,11 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager, Logge
     $this->logger = $logger;
   }
 
-  /**
-   * Gets the title of the vehicle node by its node ID.
-   *
-   * @param int $node_id
-   *   The node ID.
-   *
-   * @return string
-   *   The title of the vehicle node.
-   */
+  // Uses a node ID to get its associated title
   public function getNodeTitle($node_id){
     $node = $this->entityTypeManager->getStorage('node')->load($node_id);
     $title = $node->label();
-
+    // Logs the node ID and its title to admin/
     \Drupal::logger('vehicle_custom')->info('The following nid: @n_id was requested, and the following output returned: @n_title',[
```

Comment:
This should be using $logger. By doing \Drupal::logger, you're using the container. There shouldn't be any `\Drupal::` if you're using dependency injections